src/Workspaces/Core/Portable/Workspace/Solution/ProjectState.cs

```diff
@@ -411,6 +414,16 @@ public ProjectState UpdateOutputPath(string outputFilePath)
             return this.With(projectInfo: this.ProjectInfo.WithOutputFilePath(outputFilePath).WithVersion(this.Version.GetNewerVersion()));
         }
 
+        public ProjectState UpdateOutputRefPath(string outputRefFilePath)
+        {
+            if (outputRefFilePath == this.OutputRefFilePath)
+            {
+                return this;
+            }
+
+            return this.With(projectInfo: this.ProjectInfo.WithOutputFilePath(outputRefFilePath).WithVersion(this.Version.GetNewerVersion()));
```

Comment:
I assume you want this.ProjectInfo.WithOutput**Ref**Path here.

Reply:
Thanks! Missed that when I re-did this change on dev15.7.x.